client/src/components/SeasonCollectionImageCardComponent/seasoncollectionimg.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './seasoncollectionimg.css'
+import imageFile1 from "./Image 1.svg"
+import imageFile2 from "./Image 2.svg"
+
+
+// need to work here
+let season = "Fall";
+let year = "2020";
+
+const SeasonCollectionImageCardComponent = (props) =>
+
+    <div>
+
+        <div className="seasonCollectionSquare">
+            <div className="seasonCollectionImgContainer">
+                <img className="seasonCollectionImg" src={imageFile1} />
+                <img className="seasonCollectionImg" src={imageFile2} />
+            </div>
+            <p className="seasonCollectionP">{season} {year} Collection</p>
```

Comment:
Can you come up with more semantic names for these classes? It's generally a bad pattern to put the tag name in the class. What about something like `season-collection-text`?